### ISSUE TITLE
Documentar as sprints 0, 1 e 2

### ISSUE THREAD

**Issue by Matheuspleal** [fga-eps-mds/Projeto01#32]

**Comment by Matheuspleal** [comment]
Lembrar de organizar a documentação das sprints passadas em pastas para cada grupo!

**Comment by LeticiaAires** [comment]
@Matheuspleal @guigudf que tal fazermos uma reunião para documentarmos as sprints juntos?

**Comment by guigudf** [comment]
@LeticiaAires sim, eu tava querendo falar disso tb, eu comecei o documento na branch gh-pages, mas seria bom juntar todos os grupos num documento só

**Comment by Matheuspleal** [comment]
@LeticiaAires @guigudf Pode ser, mas para o meu grupo eu já fiz essas documentações anteriormente. Os documentos estão [aqui](https://github.com/fga-eps-mds/Projeto01/tree/main/docs/atas/grupo3).

**Comment by guigudf** [comment]
tava pensado num modelo como [esse](https://github.com/fga-eps-mds/Projeto01/blob/gh-pages/sprints/sprint2/resultado.md)  com a documentação das issues, os gráficos do zunhub, pontos positivos/negativos e um resumo do decorrer da sprint

**Comment by Matheuspleal** [comment]
Isso, a ideia é mantermos daqui pra frente a documentação desse jeitinho aí mesmo, mas acho que pra fazermos pra essas sprints passadas vai ficar difícil, pq tem bastante coisa pra fazer. Inclusive, vou passar aqui para os templates a ideia que o @gabrielm2q passou pra fazermos.

**Comment by Matheuspleal** [comment]
O template que foi sugerido está [aqui](https://github.com/fga-eps-mds/Projeto01/blob/main/docs/templates/sprintLayout.md).

**Comment by guigudf** [comment]
💡

**Comment by Matheuspleal** [comment]
@guigudf @LeticiaAires as documentações das sprints passadas já foram feitas ?

**Comment by guigudf** [comment]
@Matheuspleal sim, acabei de subir o [modelo](https://github.com/fga-eps-mds/2021.2-INDICAA/tree/main/docs/sprints), mas precisamos colocar a retrospectiva dos grupos 1 e 3 nos resultados.md

**Comment by Matheuspleal** [comment]
Como avisado antes, as atas de reunião do grupo 3 se encontram todas [aqui](https://github.com/fga-eps-mds/2021.2-INDICAA/tree/main/docs/atas/grupo3). Caso seja necessário, iremos transferir o conteúdo para o modelo proposto.

**Comment by guigudf** [comment]
> Como avisado antes, as atas de reunião do grupo 3 se encontram todas [aqui](https://github.com/fga-eps-mds/2021.2-INDICAA/tree/main/docs/atas/grupo3). Caso seja necessário, iremos transferir o conteúdo para o modelo proposto.

Certo, mudei o caminho pras [atas do grupo 4](https://github.com/fga-eps-mds/2021.2-INDICAA/tree/main/docs/atas/grupo4)
